config/arch/c6x.in

```diff
@@ -0,0 +1,37 @@
+# c6x specific configuration file
+
+## no-package
+## select ARCH_SUPPORTS_32
+## select ARCH_DEFAULT_32
+## select ARCH_SUPPORTS_EITHER_ENDIAN
+## select ARCH_DEFAULT_LE
+## select ARCH_SUPPORTS_FLAT_FORMAT
+## select ARCH_SUPPORTS_WITH_CPU
+## select TARGET_SKIP_CONFIG_SUB
```

Comment:
This was done because the "official" SDK shipped by TI had used a naming convention that wasn't even close to the canonical one.  I can move this to an example config that is tailored to working with the TI SDK.

Reply:
Removed completely and patched the SDK